_posts/2019-02-24-The-Intelligent-Investor.md

```diff
@@ -0,0 +1,152 @@
+
+  3) Do you have a safety net of between 6 - 12 months of living expenses saved up? When the shit goes down you better be ready.
+
+
+### Why Am I Focusing on Defensive Investors and not talking about Enterprising Investors?
+
+Many people try, and typically fail, to beat the market averages for many reasons, but the reality is that most people think they're above average<sup>3</sup> when in fact that's statistically impossible. Can _YOU_ beat the market consistently for decades? You can try, but it is likely that you will fail due to sheer time required, temperamental and intellectual advantage within the market, and the element of markets becoming more efficient particularly at the large capital end i.e. companies in the S&P 500.
+
+To truly beat the market over long time spans one has to have patience, extreme patience, and an ability to focus on the larger goal while keeping their emotions in check for decades. It is a slow, turbulent, and boring process for most. People like Warren Buffett, Joel Greenblatt, Charlie Munger, and Ray Dalio are not just rare, they're exceptional in ways beyond just mental powers that we can measure, like IQ.
+
+Note: I included a link at the bottom to the subject of Dysrationalia, it is a fascinating topic and one that has no received enough attention in the psychology community.
+
+If you read Vanguard's [The Case for Index Funds](https://pressroom.vanguard.com/nonindexed/Updated_The_Case_for_Index_Fund_Investing_4.9.2014.pdf) you will realize most "experts" are nothing more than highly paid money managers who on the whole detract from your portfolio returns by "managing" them when a simple index fund would beat 80%+ of these managers. This is the evil twin of compounded return, compounded loss. But wait you say, what about the other ~20+% of managers, why don't I just go with them? That group is not a consistent batch of people so how are you supposed to pick the best fund managers if they don't even know who they are?
+
+    Consider 1000 people trying to guess a coin toss.
+    Lets say half guess heads and the other half guess tails.
+    This keeps happening so 500 people are left,
+    then 250, 125, etc. until only one person is left.
+
+Is this person an oracle? Do they have some amazing insight beyond mere mortals? No, they are the last person because that's just what happens when you have so many rounds. What if you keep testing this person without others to elimination, I can almost guarantee you after a large amount of coin tosses it will become clear they cannot guess better than the average person. To extend the same logic here,
+managers who are "lucky" are no better than average most likely. They are simply "lucky" on the average. However, people who consistently show the ability to predict companies and market moments and they are rare, but to believe you are one potentially true, but highly unlikely without a track record.
```

Comment:
This reminds me of a Kurt Vonnegut short story: https://en.wikipedia.org/wiki/Report_on_the_Barnhouse_Effect

Reply:
The wiki description doesn't do it justice. The idea behind what Barnhouse discovers is how to control "luck" through his mind.

Reply:
Managers betting on coin flips is a good analogy for how managers can get lucky. 
Managers betting on how other managers bet on coin flips is a good analogy for the derivatives market.